tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -28,110 +45,426 @@ test.describe( 'Merchant who is getting started', () => {
+			await setUpAccountsPage.mockJetpackConnected(
+				'Test user',
+				'jetpack@example.com'
+			);
 
-		await expect(
-			page.getByText( 'Google', { exact: true } )
-		).toBeVisible();
+			// Mock google as not connected.
+			// When pending even WPORG will not render yet.
+			// If not mocked will fail and render nothing,
+			// as Jetpack is mocked only on the client-side.
+			await setUpAccountsPage.mockGoogleNotConnected();
 
-		expect(
-			page.getByRole( 'button', { name: 'Connect' } ).first()
-		).toBeEnabled();
+			await setUpAccountsPage.goto();
+
+			page.getByRole( 'heading', { name: 'Set up your accounts' } );
+		} );
+
+		test( 'should see their WPORG email, "Google" title & connect button', async () => {
+			const jetpackDescriptionRow =
+				setUpAccountsPage.getJetpackDescriptionRow();
+
+			await expect( jetpackDescriptionRow ).toContainText(
+				'jetpack@example.com'
+			);
+
+			await expect(
+				page.getByText( 'Google', { exact: true } )
+			).toBeVisible();
+
+			await expect(
+				page.getByRole( 'button', { name: 'Connect' } ).first()
+			).toBeEnabled();
+		} );
+
+		test( 'after clicking the "Connect your Google account" button should send an API request to connect Google account, and redirect to the returned URL', async ( {
+			baseURL,
+		} ) => {
+			// Mock google connect.
+			await setUpAccountsPage.mockGoogleConnect(
+				baseURL + 'google_auth'
+			);
+
+			// Click the enabled connect button
+			page.locator(
+				"//button[text()='Connect'][not(@disabled)]"
+			).click();
+			await page.waitForLoadState( 'networkidle' );
+
+			// Expect the user to be redirected
+			await page.waitForURL( baseURL + 'google_auth' );
+		} );
 	} );
 
-	test( 'after clicking the "Connect your Google account" button should send an API request to connect Google account, and redirect to the returned URL', async ( {
-		page,
-		baseURL,
-	} ) => {
-		// Mock google connect.
-		await page.route( /\/wc\/gla\/google\/connect\b/, ( route ) =>
-			route.fulfill( {
-				content: 'application/json',
-				headers: { 'Access-Control-Allow-Origin': '*' },
-				body: JSON.stringify( {
-					url: baseURL + 'google_auth',
-				} ),
-			} )
-		);
-
-		// Click the enabled connect button
-		page.locator( "//button[text()='Connect'][not(@disabled)]" ).click();
-		await page.waitForLoadState( 'networkidle' );
-
-		// Expect the user to be redirected
-		await page.waitForURL( baseURL + 'google_auth' );
+	test.describe( 'Connect Merchant Center account', () => {
+		test.beforeAll( async () => {
+			await Promise.all( [
+				// Mock Jetpack as connected.
+				setUpAccountsPage.mockJetpackConnected(
+					'Test user',
+					'jetpack@example.com'
+				),
+
+				// Mock google as connected.
+				setUpAccountsPage.mockGoogleConnected( 'google@example.com' ),
+
+				// Mock merchant center as not connected.
+				setUpAccountsPage.mockMCNotConnected(),
+			] );
+		} );
+
+		test.describe( 'Create a new Merchant Center account', () => {
```

Comment:
💅  I would change this to "Merchant Center has no existing accounts" 

If I understand it well, its more accurate with this test case